saleor/order/actions.py

```diff
@@ -33,14 +33,30 @@ def order_created(order: "Order", user: "User", from_draft: bool = False):
     events.order_created_event(order=order, user=user, from_draft=from_draft)
     manager = get_plugins_manager()
     manager.order_created(order)
+    handle_post_checkout_events(order=order, user=user)
 
 
-def handle_fully_paid_order(order: "Order"):
-    events.order_fully_paid_event(order=order)
+def handle_post_checkout_events(order: "Order", user: "User"):
+    payment = order.get_last_payment()
+    if payment:
+        if order.is_captured():
+            order_captured(
+                order=order, user=user, amount=payment.total, payment=payment
+            )
+        elif order.is_pre_authorized():
+            order_authorized(
+                order=order, user=user, amount=payment.total, payment=payment
+            )
+        if order.is_fully_paid():
+            handle_fully_paid_order(order=order, user=user)
```

Comment:
This function is also called in `gateway_postprocess`. Wouldn't calls to this function be duplicated in some flows? Ideally, we should remove it from `gateway_postprocess`.